eventsource/stores/sqlstore/store_test.go

```diff
@@ -197,5 +216,5 @@ CREATE TABLE %s (
 	assert.Nil(t, err, "Could not get events")
 	assert.Equal(t, 1, len(events))
 
-	db.Exec("DROP TABLE $1", tmpTableName)
+	db.Exec(fmt.Sprintf("DROP TABLE IF EXISTS %s", tmpTableName))
```

Comment:
Shouldn't this run as a defer after table is created?

Reply:
Yes, that would probably be better